data/json/monsters/zed_amalgamation.json

```diff
@@ -0,0 +1,331 @@
+    "attack_cost": 100,
+    "melee_dice": 2,
+    "melee_dice_sides": 4,
+    "melee_damage": [ { "damage_type": "cut", "amount": 5, "armor_penetration": 5 } ],
+    "special_attacks": [
+      {
+        "type": "leap",
+        "min_consider_range": 2,
+        "min_range": 2,
+        "move_cost": 50,
+        "max_range": 8,
+        "allow_no_target": false,
+        "cooldown": 3,
+        "forbidden_effects_any": [ "maimed_leg", "maimed_wings" ]
+      },
+      {
+        "type": "bite",
+        "cooldown": 3,
+        "move_cost": 150,
+        "damage_max_instance": [ { "damage_type": "cut", "amount": 15, "armor_penetration": 8 } ],
+        "forbidden_effects_any": [ "maimed_mandible" ],
+        "infection_chance": 10
+      }
+    ],
+    "dodge": 7,
+    "extend": { "flags": [ "HARDTOSHOOT" ], "weakpoint_sets": [ "wps_amalgamation_wing", "wps_amalgamation_mandible" ] }
+  },
+  {
+    "type": "MONSTER",
+    "id": "mon_amalgamation_zapper",
+    "copy-from": "mon_amalgamation_abstract_med",
+    "name": { "str": "charged amalgamation" },
+    "description": "A short, crablike creature waddling around on six multi-jointed legs, its body dominated by a large glowing organ bulging from between the white armor plates. A single raised appendage connects to the mass, its tip surrounded by brilliant white sparks.",
+    "color": "blue",
+    "speed": 95,
+    "luminance": 8,
+    "melee_damage": [ { "damage_type": "electric", "amount": 5 } ],
+    "special_attacks": [
+      {
+        "type": "spell",
+        "id": "shock_blast",
+        "spell_data": { "id": "shock_blast", "hit_self": true, "min_level": 1 },
+        "cooldown": 25,
+        "monster_message": "A cloud of electricity explodes from the %s",
+        "required_effects_any": [ "elec_charged" ],
+        "forbidden_effects_any": [ "maimed_capacitor" ]
+      },
+      {
+        "type": "spell",
+        "id": "shock_cone",
+        "spell_data": { "id": "shock_cone", "min_level": 1 },
+        "monster_message": "A bolt of electricity arcs from %s!",
+        "cooldown": 20,
+        "forbidden_effects_any": [ "maimed_capacitor" ]
+      },
+      { "id": "elec_charge", "forbidden_effects_any": [ "maimed_capacitor" ] }
+    ],
+    "special_when_hit": [ "ZAPBACK", 100 ],
+    "death_function": { "effect": { "id": "death_shock_blast", "hit_self": true } },
+    "harvest": "zombie_amalgamation_electric",
+    "extend": { "flags": [ "ELECTRIC" ], "weakpoint_sets": [ "wps_amalgamation_armor", "wps_amalgamation_electric" ] }
+  },
+  {
+    "type": "MONSTER",
+    "id": "mon_amalgamation_soldier",
+    "copy-from": "mon_amalgamation_abstract_med",
+    "name": { "str": "brutish amalgamation" },
+    "description": "A large, thickly armored creature built not unlike a bear, if a bear had a long, clubbed tail and two short limbs ending in black talons framing a vertical mouth on an otherwise smooth, featureless head.  On second though, not that much like a bear at all.",
```

Comment:
```suggestion
    "description": "A large, thickly armored creature built not unlike a bear, if a bear had a long, clubbed tail and two short limbs ending in black talons framing a vertical mouth on an otherwise smooth, featureless head.  On second thought, not that much like a bear at all.",
```